### PR TITLE
Editor for zoomed in view: normal blocks and tasks

### PR DESCRIPTION
**Pull request by sid597**

### CODE REVIEW

File: src/cljs/athens/types/default/view.cljs [hunk 1]
```diff
@@ -189,7 +191,56 @@
 
 
   (zoomed-in-view
-    [_this _block-data _callbacks])
+    [_this block-data callbacks]
+    (let [{:block/keys [uid
```
Comment (neotyk) [inline src/cljs/athens/types/default/view.cljs:195]: Please move it to dedicated `defn` above protocol definition.
When it's inline, protocol implementation grows big very fast.

File: src/cljs/athens/types/default/view.cljs [hunk 2]
```diff
@@ -189,7 +191,56 @@
...
+        [_this block _callbacks]
+        (let [ident                 [:block/uid (or original-uid uid)]
+              block-o               (reactive/get-reactive-block-document ident)
+              {:block/keys [string
+                            _refs]} (merge block-o block)]
+
+          ;; If datascript string value does not equal local value, overwrite local value.
+          ;; Write on initialization
+          ;; Write also from backspace, which can join bottom block's contents to top the block.
+          (when (not= string @old-value)
```
Comment (neotyk) [inline src/cljs/athens/types/default/view.cljs:239]: This is a source of bugs that we had where we couldn't backspace in Assignee.
Just always execute it.

File: src/cljs/athens/types/tasks/view.cljs [hunk 3]
```diff
@@ -662,7 +662,54 @@
 
 
   (zoomed-in-view
-    [_this _block-data _callbacks])
+    [_this block-data _callbacks]
+    (let [parent-block-uid    (:block/uid block-data)
```
Comment (neotyk) [inline src/cljs/athens/types/tasks/view.cljs:666]: Same as in default implementation.

File: src/cljs/athens/views/right_sidebar/shared.cljs [hunk 4]
```diff
@@ -111,5 +111,5 @@
                :children   (r/as-element (cond
                                            (= type "graph") [graph/page name]
                                            (= type "page") [node-page/page eid]
-                                           :else [block-page/page eid]))}))
+                                           :else [blocks/page eid]))}))
```
Comment (neotyk) [inline src/cljs/athens/views/right_sidebar/shared.cljs:114]: We should check for BlockTypeProtocol availability and use it here.

File: src/cljs/athens/views/pages/page.cljs [hunk 5]
```diff
@@ -29,5 +30,5 @@
     [:> PageContainer {:uid @uid :type (if title "node" "block")}
      (cond
        title [node-page/page id]
-       string [block-page/page id]
+       string [blocks/page id]
```
Comment (neotyk) [inline src/cljs/athens/views/pages/page.cljs:33]: We should check for BlockTypeProtocol availability and use it here.

**Review by neotyk — CHANGES_REQUESTED** [review]

File: src/cljs/athens/views/right_sidebar/shared.cljs (hunk 4 above)
Comment (sid597) [reply src/cljs/athens/views/right_sidebar/shared.cljs:114]: You mean this `renderer-k                 (block-type-dispatcher/block-type->protocol-k block-type ff)`? 
I am using it in the block-page-el.

File: src/cljs/athens/types/default/view.cljs (hunk 2 above)
Comment (sid597) [reply src/cljs/athens/types/default/view.cljs:239]: Should I remove this from `outline view` also, I copied this piece from there?

**Review by neotyk — APPROVED** [review]
GG WP

**Pull request merged**